metricbeat/beater/module.go

```diff
@@ -0,0 +1,204 @@
+package beater
```

Comment:
The most recent case I have in mind is the refactoring of Topbeat. The event building was also part of the beater and it had to be extracted to be used in topbeat and metricbeat. So who knows :-) Definitively a step we can take when its needed.